examples/cp.py

```diff
@@ -0,0 +1,94 @@
+"""
+Check that the CP calculation might not work.
+"""
+
+from __future__ import print_function
+
+from pprint import pprint
+
+from aiida.engine import run
+from aiida.orm import Code, Dict, KpointsData, StructureData
+from aiida.orm.nodes.data.upf import get_pseudos_from_structure
+from aiida.plugins.factories import CalculationFactory
+from ase.spacegroup import crystal
+
+
+def main():
+    """
+    Little check on the cp calculation thing.
+    """
+    alat = 5.4
+    ase_structure = crystal(
+        "Si",
+        [(0, 0, 0)],
+        spacegroup=227,
+        cellpar=[alat, alat, alat, 90, 90, 90],
+        primitive_cell=True,
+    )
+    structure = StructureData(ase=ase_structure)
+    structure.store()
+
+    calculation = CalculationFactory("quantumespresso.cp")
+    builder = calculation.get_builder()
+
+    # get some code
+    code = Code.objects.all()[0]
```

Comment:
Note that this will horribly fail of course unless the first code so happens to be configured to run `cp.x`. If this a testing script just for yourself, I'd rather not put it in the repository (in any case this `example` directory is so old and outdated, we should remove it). If we want to make it more generally usable we could consider adding a launcher, however, we should think if we really want to add one for each code.

Reply:
What's the problem with adding a launcher for every code? Just the effort required?

Reply:
The "problem" is that this adds an executable for each launcher in the environment of the user. Having a few may be ok, but polluting the space with dozens of executables may not be desirable. If we do want to support multiple launchers, we may want to refactor them to use `click` and its group functionality to create one end point, that allows the other commands to be reached through subcommands, e.g. `aiida-quantumespresso launch calculation pw --help`

Reply:
I see. Makes sense.